index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="assets/css/style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="main-header">
+        <h1 class="logo">My Logo</h1>
+        <nav class="header-links-phone">
+            <img class="btn" src="assets/icons/menu.png" width="25px" height="25px" alt="">
+        </nav>
+        <nav class="header-links-desktop">
+            <ul>
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+    <section class="main-section">
+        <div class="div-description">
+            <h2>I'm Bruno.<br />Glad to see you!</h1>
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+                experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        </div>
+        <div class="div-connect">
+            <p>LET’S CONNECT</p>
```

Comment:
:red_circle: I think this looks more like a title than a paragraph. It should use a header tag hx (examples: `<h4>,<h3>,<h2>,` etc...). What do you think :thinking:

Reply:
You're right, thanks @bienvenuushindi